external/miniz/Kconfig

```diff
@@ -3,7 +3,7 @@
 # see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
 #
 
-config COMPRESSION_MINIZ
+config LIBMINIZ
 	bool "Miniz Compression"
 	default y if COMPRESSION_TYPE=2 && BUILD_PROTECTED=n
```

Comment:
Don't you need to delete this line? Could you let me know why you does for LZMA only?

Reply:
There is some dependency of these flags with app/common binary compression, I will remove them in new PR with miniz cleanup